h2o-r/h2o-package/R/models.R

```diff
@@ -3473,8 +3473,6 @@ row_index=-1) {
   if (!noPairs &&
       ("plot3Drgl" %in% rownames(installed.packages())) &&
       ("rgl" %in% rownames(installed.packages()))) {
```

Comment:
please generate a warning when either of these packages is not available. the warning should instruct users what specifically they need to do in order to enable this functionality

Reply:
Done.